utils/generator/templates/struct_function_template.java

```diff
@@ -0,0 +1,69 @@
+{% extends "base_template.java" %}
```

Comment:
This template has some issues with generating responses. All the responses are generated without any getter or setters.

Reply:
there was redundant check:
```jinja
    {%- if params is defined and ((kind is defined and kind != "response") or kind is not defined) %}
```
After deleting works well

Reply:
But in original RPC library all function responses does not have any getters/setters as well as parameters list. Only requests have getters/setters and parameters list. For responses parameters "success" and "resultCode" defined in RPCResponse.

Reply:
The shared getters/ setters are placed in the base RPCResponse class. But also each response has its own getters and setters. For example:
https://github.com/smartdevicelink/sdl_java_suite/blob/797b7a14a7fd67fdf6faa504865e4e22228858ef/base/src/main/java/com/smartdevicelink/proxy/rpc/GetVehicleDataResponse.java#L108

Reply:
Got it, thanks, working on it